src/Umbraco.Infrastructure/PropertyEditors/ValueListConfigurationEditor.cs

```diff
@@ -18,7 +21,20 @@ namespace Umbraco.Cms.Core.PropertyEditors
     /// </remarks>
     public class ValueListConfigurationEditor : ConfigurationEditor<ValueListConfiguration>
     {
-        public ValueListConfigurationEditor(ILocalizedTextService textService, IIOHelper ioHelper) : base(ioHelper)
+        // Scheduled for removal in v12
+        [Obsolete("Please use constructor that takes an IEditorConfigurationParser instead")]
+        public ValueListConfigurationEditor(ILocalizedTextService textService, IIOHelper ioHelper)
+         : this(textService, ioHelper, StaticServiceProvider.Instance.GetRequiredService<IEditorConfigurationParser>())
+        {
+            var items = Fields.First(x => x.Key == "items");
+
+            // customize the items field
+            items.Name = textService.Localize("editdatatype", "addPrevalue");
+            items.Validators.Add(new ValueListUniqueValueValidator());
+        }
```

Comment:
```suggestion
        [Obsolete("Please use constructor that takes an IEditorConfigurationParser instead")]
        public ValueListConfigurationEditor(ILocalizedTextService textService, IIOHelper ioHelper)
         : this(textService, ioHelper, StaticServiceProvider.Instance.GetRequiredService<IEditorConfigurationParser>())
        {
        }
```

Since we're invoking the other constuctor with `this` I don't we have to do this again here.